### PR TITLE
Refactor counts logging

### PR DESCRIPTION
**Pull request by pravinbhat**

### CODE REVIEW

File: src/main/java/datastax/astra/migrate/CopyJobSession.java [hunk 1]
```diff
@@ -126,13 +123,24 @@ public void getDataAndInsert(BigInteger min, BigInteger max) {
         }
     }
 
+    public synchronized void printCounts(boolean isFinal) {
+        String msg = "TreadID: " + Thread.currentThread().getId();
```
Comment (msmygit) [inline src/main/java/datastax/astra/migrate/CopyJobSession.java:127]: #nitpick #trivial I would change `TreadID` to `ThreadID` on this typo?

File: src/main/java/datastax/astra/migrate/CopyJobSession.java [hunk 2]
```diff
@@ -126,13 +123,24 @@ public void getDataAndInsert(BigInteger min, BigInteger max) {
...
+        if (isFinal) {
+            msg += " Final";
+            logger.info("################################################################################################");
+        }
+        logger.info(msg + " Read Record Count: " + readCounter.get());
```
Comment (msmygit) [inline src/main/java/datastax/astra/migrate/CopyJobSession.java:132]: #nitpick I'd change this to use substitutions as opposed to concatinations for efficiency like,

```
logger.info("{} Read Record Cont: {}", msg, readCounter.get());
```

**Review by msmygit — APPROVED** [review]
LGTM. Left very minor/trivial comments.

File: src/main/java/datastax/astra/migrate/CopyJobSession.java (hunk 1 above)
Comment (pravinbhat) [reply src/main/java/datastax/astra/migrate/CopyJobSession.java]: @msmygit Great inputs. Can you please review one more time. Thx!

**Pull request merged**